src/pages/HomePage.jsx

```diff
@@ -1,6 +1,6 @@
 import { Layout } from '../components';
 import { Signup } from '../components';
-import { Input } from '../components/inputs/Input';
+import { Input } from '../components';
```

Comment:
One import is enough
import {Input, Signup, Layout} from "../components"

Reply:
fixed